example/ipfragreass/Makefile.am

```diff
@@ -1,5 +1,7 @@
 include $(top_srcdir)/example/Makefile.inc
 
+LDADD += $(ATOMIC_LIBS)
```

Comment:
This kind of thing which is better to avoid. If we are going to merge odp implementations platforms to common repo then it might be libs which are suitable for linux-generic are not needed for other platform. I think examples Makefiles have to be more common.

Reply:
This example is not generic enough. It uses atomic primitives directly.

Reply:
Ok, Than. Agree.